modules/installation-methods-for-different-architectures.adoc

```diff
@@ -14,10 +14,13 @@
 |*Supported architectures*|*Supported installation methods*
 
 |AMD64
-a|Operator (preferred), Helm charts, or `roxctl` CLI (not recommended)
+a|Operator (preferred), Helm charts, `roxctl` CLI (not recommended)
 
 | ppc64le ({ibm-power})
 .2+a|Operator
 
 | s390x ({ibm-z} and {ibm-linuxone})
+
+|Azure Resource Manager (ARM)
+|Helm charts (recommended), `roxctl` CLI
```

Comment:
We probably want Operator here as well, I just didn't have time to test it, but it is supported.